image_width: 640
image_height: 480
camera_name: camera_low_light_left
camera_matrix:
  rows: 3
  cols: 3
  data: [613.12164,   0.     , 305.40464,
           0.     , 610.5443 , 254.37699,
           0.     ,   0.     ,   1.     ]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.348078, 0.261385, 0.000313, -0.013205, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [ 0.95407683, -0.03047623, -0.29800773,
          0.02636285,  0.99949371, -0.01781367,
          0.29839975,  0.00913927,  0.95439723]
projection_matrix:
  rows: 3
  cols: 4
  data: [796.01671,   0.     , 529.49941,   0.     ,
           0.     , 796.01671, 267.99264,   0.     ,
           0.     ,   0.     ,   1.     ,   0.     ]
